py3dtilers/GeojsonTiler/README.md

```diff
@@ -168,4 +168,24 @@ Project the features on another CRS. The `crs_in` flag allows to specify the inp
 geojson-tiler --path <path> --crs_in EPSG:3946 --crs_out EPSG:4171
 ```
 
-### With texture
+### Geometric error
```

Comment:
This looks a identical to the eponymous chapter of CityTiler/README.md (of this PR).
Apply DRY (Don't Repeat Yoursel) and introduce a common/shared flags section in the documention and then point to it.

Reply:
It will be done once both this PR and [CityTiler colors PR](https://github.com/VCityTeam/py3dtilers/pull/82) are closed (see [issue](https://github.com/VCityTeam/py3dtilers/issues/83))